ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -231,6 +232,16 @@ do
   yq-go write -i "${CONFIG}" "platform.aws.userTags.${TAG}" "${VALUE}"
 done
 
+if [[ "${PROPAGATE_USER_TAGS:-}" == "yes" ]]; then
```

Comment:
How about developing a separate step to do the operation?

Reply:
`PROPAGATE_USER_TAGS` is used along with `USER_TAGS`, I thought we use the same logic to handle them, if we create a separate step, we also need to extract USER_TAGS together.

Reply:
got it.